.vitepress/theme/components/header/Navigation.vue

```diff
@@ -9,6 +9,10 @@ import ThemeSwitcher from './ThemeSwitcher.vue'
 defineProps<{ navigation: NavigationType }>()
 
 const currentTrigger = ref('')
+
+function getTarget(url: string) {
+  return url.startsWith('http') ? '_blank' : ''
```

Comment:
When you return `undefined` as the _else_ case, does it leave out the `target` attribute then in the rendered HTML?